scene/3d/audio_stream_player_3d.cpp

```diff
@@ -240,6 +240,7 @@ float AudioStreamPlayer3D::_get_attenuation_db(float p_distance) const {
 }
 
 void AudioStreamPlayer3D::_notification(int p_what) {
+	internal->notification(p_what);
 	switch (p_what) {
```

Comment:
```suggestion
	internal->notification(p_what);

	switch (p_what) {
```

If we wanted to be more consistent with AudioStreamPlayer2D.

Reply:
I tend to avoid empty lines that do nothing, also it's more consistent with this code:

```cpp
void AudioStreamPlayer3D::_calc_output_vol(const Vector3 &source_dir, real_t tightness, Vector<AudioFrame> &output) {
	unsigned int speaker_count = 0; // only main speakers (no LFE)
	switch (AudioServer::get_singleton()->get_speaker_mode()) {
```